samples/nrf9160/lte_ble_gateway/prj.conf

```diff
@@ -65,6 +73,7 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 # Heap and stacks
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 CONFIG_MAIN_STACK_SIZE=8192
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
```

Comment:
Why is it necessary to override the default now?

Reply:
To avoid stack overflow when processing AGPS data.